README.rst

```diff
@@ -47,6 +47,7 @@ Configuration options for Thoth's s2i image:
 
 * ``THOTH_ADVISE`` - always use the recommended stack by Thoth (even if the lock file is present in the repo)
 * ``THOTH_CONFIG_CHECK`` - verify values stated in the configuration file match the build environment
+* ``THOTH_GENERATE_CONFIG`` - generate configuration file during the build process, overwrites already existing configuration file if present
```

Comment:
this feels more like `THOTH_FORCE_GENERATION_OF_CONFIG` shall we make the overwrite more explicit to the user?